onprc_ehr/resources/data/lookup_sets.tsv

```diff
@@ -43,6 +43,7 @@ alopecia_type	Alopecia Type	value
 alopecia_score	Alopecia Type	value
 alopecia_regrowth	Alopecia Regrowth	value
 mens_score	Mens Score	value
+vomit   Vomit   value
```

Comment:
The whitespace between these values needs to be tabs. You have spaces here.

Reply:
@labkey-martyp, I have made that correction and automated tests still failed with the same errors.

Reply:
@kollil  Marty's requested changes were not implemented although it looks like you found another place where tabs should be used. Marty was talking abou thte lookup_sets.tsv file. To demonstrate the issue, download the tsv and open it in excel. https://github.com/LabKey/onprcEHRModules/blob/23.11_fb_VomitObs/onprc_ehr/resources/data/lookup_sets.tsv

Please notice that this comment thread start by Marty is pointing at the lookup_sets.tsv file, not the vomit.tsv file.

Reply:
@jryurkanin, Thanks for giving me the tip of downloading into an excel. I did add the tabs in the code first time Marty pointed out, but, somehow the tabs were not getting inserted in the code. I copied the entire tsv file into wordpad, added tabs and then pasted back into the code. Finally it worked.

Reply:
Glad that worked for you. I actually recommend building the TSV files in excel, then you can copy and paste the table from excel straight into the github website code editor. When you do that, it retains the correct tabular format from excel.